### PR TITLE
rswift 5.1.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
[34m==>[0m [1mbrew test rswift --verbose[0m
Testing rswift
/usr/bin/sandbox-exec -f /private/tmp/homebrew20200412-21392-gd49kd.sb ruby -W0 -I $LOAD_PATH -- /usr/local/Homebrew/Library/Homebrew/test.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/rswift.rb --verbose
[34m==>[0m [1m/usr/local/Cellar/rswift/5.1.0/bin/rswift --version[0m
dyld: Library not loaded: @rpath/libswiftCore.dylib
  Referenced from: /usr/local/Cellar/rswift/5.1.0/bin/rswift
  Reason: image not found

[31mError:[0m rswift: failed
An exception occurred within a child process:
  Test::Unit::AssertionFailedError: <0> expected but was
<nil>.
```

**Comment by chenrui333** [comment]
relates to https://github.com/Homebrew/homebrew-core/pull/52584

**Comment by Bo98** [comment]
Try remove the `-Xswiftc -static-stdlib`

**Comment by chenrui333** [comment]
@Bo98 that works!

File: Formula/rswift.rb [hunk 1]
```diff
@@ -15,8 +15,7 @@ class Rswift < Formula
   depends_on :xcode => "10.0"
```
Comment (Bo98) [inline Formula/rswift.rb:15]:
```suggestion
  depends_on :xcode => "10.2"
```

**Review by Bo98 — COMMENTED** [review]
One more thing:

**Comment by chenrui333** [comment]
rebased to the latest master

**Pull request closed without merging**